### DISCUSSION TITLE
🙏 Please add compress video option

### DISCUSSION THREAD

**Discussion by Prasath-sk-14188** [mifi/lossless-cut#855] [Ideas]
I know its contradicting to the main purpose of the tool.. The tool is so conveniant that, I use Premiere pro only for compressing videoes. I t will be nice to add a simple video compression option in it.

**Comment by mifi** [comment]
can't you use handbrake? or do you want to cut *and* compress in the same action?

**Reply by Prasath-sk-14188** [reply]
yeah, I can do that. I would like it if I have an option to compress it while exporting

**Reply by mifi** [reply]
leave a like at #372
